src/components/Filters/FilterByClothingType/FilterByClothingType.spec.js

```diff
@@ -0,0 +1,18 @@
+import { render, screen } from '@testing-library/react'
+import userEvent from '@testing-library/user-event'
+
+import FilterByInputClothing from './FilterByInputClothing'
+
+describe('FilterByInputClothing', () => {
+  it('renders an select with the options e.g. "Kleider" and the Callback onChange', () => {
+    const callback = jest.fn()
+    render(
+      <FilterByInputClothing
+        onChange={callback}
```

Comment:
See post in Slack about functions which aren't props: https://neuefische-students.slack.com/archives/G01FT9RC8FQ/p1616400319016500
You don't have a prop `onChange` in this component. 
```suggestion

```